scripts/package/fpsci_packager.py

```diff
@@ -0,0 +1,68 @@
+# This script generates a packaging script based on the files used by the last successful run of the program.
+# You need to run the generated script to create the distribution directory, then zip that up to send to people.
+
+# Name of the packaging script to generate
+outputScript = 'package.sh'
+
+# Path of the log.txt file to use
+inputLog = 'data-files/log.txt'
```

Comment:
The input log file and output script name could be (optional) command arguments, but not required.

Reply:
Let's make command line options a future addition.